### DISCUSSION TITLE
Web Version?

### DISCUSSION THREAD

**Discussion by Webfeger** [andycb/AdventurerClientJS#72] [Ideas]
Hello,
it's possible to convert this app into a web version? I would run this application on a web server like octopirint

Best regards,
Web

**Comment by andycb** [comment]
Hey, 

It should be possible - the bulk of the app is an Angular web app. It uses some Electron/NodeJS APIs for the socket communication, but I believe there are browser equivalents. It also uses Electron to write some error logs to the file system, which would need to be worked around. 

Andy
